docs/Migrating-v3.md

```diff
@@ -50,6 +50,7 @@ The following packages have been re-organized:
 - `@ecl/polyfills` has been moved to `@ecl/dom-utils/polyfills`
 - `@ecl/(ec|eu)-auto-init` have been moved to `@ecl/dom-utils/autoinit`
 - `@ecl/(ec|eu)-base/helpers` have been moved to `@ecl/dom-utils`
+- all generic form packages (feedback-message, form-group, help-block, form-label) have been merged into `@ecl/vanilla-component-form`
```

Comment:
Thanks for adding this note in the document, it's useful to communicate

Please move the note under `@ecl/ec-utility-text has been removed` or some other line as otherwise adding it here breaks the logical flow of the examples for the `@ecl/dom-utils` which follow under it.

Not a blocker, but good to have, replace "all generic form packages" with their corresponding items to follow the pattern in the list

`@ecl/(ec|eu)-component-form-form-group`, `@ecl/(ec|eu)-component-form-feedback-message`, `@ecl/(ec|eu)-component-form-help-block` and `@ecl/(ec|eu)-component-form-form-label` have been consolidated in `@ecl/vanilla-component-form`